files/galaxy/tpv/destinations.yml.j2

```diff
@@ -541,8 +541,15 @@ destinations:
     env:
       GPU_AVAILABLE: 1
     params:
-      requirements: 'GalaxyGroup == "training-pxe-test-gpu"'
+      requirements: 'GalaxyGroup == "pxe-gpu"'
+      request_gpus: "{gpus or 0}"
+      docker_run_extra_arguments: "--gpus all --env CUDA_VISIBLE_DEVICES=$_CONDOR_AssignedGPUs --env NVIDIA_VISIBLE_DEVICES=$_CONDOR_AssignedGPUs"
```

Comment:
Having `docker_run_extra_arguments` here will override any additional run arguments set at the tool level. So, it is wise to reconsider this choice and adapt to something like what was discussed [here](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/1554#issuecomment-2994957093)

Reply:
I added it but I think the CI fails now, I think this is because jinja thinks it is an ansible variable.
I tried with a single curly brace

Reply:
I am not entirely sure about the single curly brace. Let's try and if you observe a different behavior, we can inspect.